src/event-bridge/event-bus.ts

```diff
@@ -349,27 +437,95 @@ export class EventBus<E extends EventBusRuleInput> extends EventBusBase<E> {
    * new functionless.EventBus.fromBus(awsBus);
    * ```
    */
-  static default<E extends EventBusRuleInput>(stack: Stack): IEventBus<E>;
-  static default<E extends EventBusRuleInput>(scope: Construct): IEventBus<E>;
-  static default<E extends EventBusRuleInput>(
+  static default<E extends EventBusEvent>(stack: Stack): DefaultEventBus<E>;
+  static default<E extends EventBusEvent>(scope: Construct): DefaultEventBus<E>;
+  static default<E extends EventBusEvent>(
     scope: Construct | Stack
-  ): IEventBus<E> {
+  ): DefaultEventBus<E> {
+    return new DefaultEventBus<E>(scope);
+  }
+
+  /**
+   * Creates a schedule based event bus rule on the default bus.
+   *
+   * Always sends the {@link ScheduledEvent} event.
+   *
+   * ```ts
+   * const bus = EventBus.default(scope);
+   * // every hour
+   * const everyHour = bus.schedule(scope, 'cron', aws_events.Schedule.rate(Duration.hours(1)));
+   *
+   * const func = new Function(scope, 'func', async (payload: {id: string}) => console.log(payload.id));
+   *
+   * everyHour
+   *    .map(event => ({id: event.id}))
+   *    .pipe(func);
+   * ```
+   */
+  static schedule(
+    scope: Construct,
+    id: string,
+    schedule: aws_events.Schedule,
+    props?: Omit<
+      aws_events.RuleProps,
+      "schedule" | "eventBus" | "eventPattern" | "targets"
+    >
+  ): ImportedRule<ScheduledEvent> {
+    return EventBus.default(scope).schedule(scope, id, schedule, props);
+  }
+}
+
+export class DefaultEventBus<E extends EventBusEvent> extends EventBusBase<E> {
+  constructor(scope: Construct) {
     const stack = scope instanceof Stack ? scope : Stack.of(scope);
     const bus =
       (stack.node.tryFindChild(
-        EventBus.#singletonDefaultNode
+        EventBusBase.singletonDefaultNode
       ) as aws_events.IEventBus) ??
       aws_events.EventBus.fromEventBusName(
         stack,
-        EventBus.#singletonDefaultNode,
+        EventBusBase.singletonDefaultNode,
         "default"
       );
 
-    return EventBus.fromBus<E>(bus);
+    super(bus);
+  }
+
+  /**
+   * Creates a schedule based event bus rule on the default bus.
+   *
+   * Always sends the {@link ScheduledEvent} event.
+   *
+   * ```ts
+   * // every hour
+   * const everyHour = EventBus.schedule(scope, 'cron', aws_events.Schedule.rate(Duration.hours(1)));
```

Comment:
I wonder if we could have a simpler interface on top?

```ts
Schedule.every(1, "minute").pipe(f)
```

Reply:
Later? This would just be a helper? plus there are the other forms, `cron` and `expression`.

https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-create-rule-schedule.html#eb-rate-expressions

Reply:
https://github.com/functionless/functionless/issues/188